### PR TITLE
BXMSDOC-629 Explain steps needed to configure SSH on server to use RSA

### PR DESCRIPTION
**Pull request by emmurphy1**
[RHPAM draft](http://file.ork.redhat.com/~emmurphy/BXMSDOC-629-pam/#ssh-configuring-proc_install-on-eap)
[RHDM draft](http://file.ork.redhat.com/~emmurphy/BXMSDOC-629-dm/#ssh-configuring-proc_install-on-eap)

### CODE REVIEW

File: doc-content/enterprise-only/installation/ssh-configuring-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ 
+* Include the `-Dorg.uberfire.nio.git.ssh.algorithm=RSA` parameter when you start {CENTRAL}, for example:
++
+[source]
+----
+$ ./standalone.sh -c standalone-full.xml 
+-Dorg.uberfire.nio.git.ssh.algorithm=RSA
+----
+
+= Verifying the {PRODUCT} installation
```
Comment (kaldesai) [inline doc-content/enterprise-only/installation/ssh-configuring-proc.adoc:37]: In addition to **= Configuring SSH to use RSA** , this is the new chapter in the doc right?

**Review by kaldesai — APPROVED** [review]
Everything looks good to me.

File: doc-content/enterprise-only/installation/ssh-configuring-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,86 @@
+[id='ssh-configuring-proc_{context}']
+
+= Configuring SSH to use RSA
+SSH is used to clone Git repositories. By default, the DSA encryption algorithm is provided by Business Central. However, some SSH clients, for example SSH clients in the Fedora 23 environment, use the RSA algorithm instead of the DSA algorithm. Business Central contains a system property that you can use to switch from DSA to RSA if required. 
+
+[NOTE]
+====
+SSH clients on supported systems, for example Red Hat Enterprise Linux 7, are not affected by this issue.
```
Comment (domhanak) [inline doc-content/enterprise-only/installation/ssh-configuring-proc.adoc]: Could we add a link to all supported systems or is it an overkill?

Comment (emmurphy1) [reply doc-content/enterprise-only/installation/ssh-configuring-proc.adoc]: I'm not aware of a link to 6.4 supported systems, just 7.x.

File: doc-content/enterprise-only/installation/ssh-configuring-proc.adoc (hunk 1 above)
Comment (emmurphy1) [reply doc-content/enterprise-only/installation/ssh-configuring-proc.adoc:37]: Yes

File: doc-content/enterprise-only/installation/ssh-configuring-proc.adoc (hunk 2 above)
Comment (emmurphy1) [reply doc-content/enterprise-only/installation/ssh-configuring-proc.adoc]: But I did add the link for the 7.x books.

**Pull request merged**